config/application.rb

```diff
@@ -89,6 +89,9 @@ class Application < Rails::Application
       'Feature-Policy' => "accelerometer 'none'; ambient-light-sensor: 'none', autoplay: 'none', battery: 'none', camera 'none'; display-capture: 'none', document-domain: 'none', fullscreen: 'none', geolocation 'none'; gyroscope 'none'; magnetometer 'none'; microphone 'none'; midi: 'none', payment 'none'; publickey-credentials-get: 'none', usb 'none', wake-lock: 'none', screen-wake-lock: 'none', web-share: 'none'",
     }
 
+    config.active_record.legacy_connection_handling = false
```

Comment:
this is part of the defaults in 6.1, can we do this when we set the defaults on then?